app/controllers/entries_controller.rb

```diff
@@ -148,4 +129,22 @@ def undo
 			}
 		end
 	end
+
+	def destroy_entries
+		begin
+			dictionary = Dictionary.editable(current_user).find_by_name(params[:dictionary_id])
+			raise ArgumentError, "Could not find the dictionary" if dictionary.nil?
+
+			raise ArgumentError, "No entry to be deleted is selected" unless params[:entry_id].present?
+
+			entries = Entry.where(id: params[:entry_id])
+			entries.destroy_all
```

Comment:
```suggestion
			Entry.where(id: params[:entry_id]).destroy_all
```

entries変数は他に使っていません。
変数を削除して、1行にまとめてよいと思います。